### PR TITLE
Proofing reading, grammar & syntax corrections made to all docs

### PR DESCRIPTION
**Pull request by eliotis**
Your work is very much appreciated and that's why I put in the time to make the documents as professional  sounding as possible. I have plans to convert all of the docs - once you approve them - into mp3 files so I listen to them while driving etc. to absorb the info even more. Thanks again and please contact me if you need to add more docs etc. so I can proof them. It's been a pleasure!
Takis

### CODE REVIEW

**Comment by itstimetogoleft** [comment]
The changes look great to me! Incredible work @eliotis

File: README.md [hunk 1]
```diff
@@ -9,15 +9,15 @@
 [![install size](https://packagephobia.now.sh/badge?p=type-graphql)](https://packagephobia.now.sh/result?p=type-graphql)
 [![gitter](https://badges.gitter.im/type-graphql.svg)](https://gitter.im/type-graphql?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Create GraphQL schema and resolvers with TypeScript, using classes and decorators!
+Create GraphQL schemata and resolvers with TypeScript, using classes and decorators!
```
Comment (itstimetogoleft) [inline README.md]: schemata?

Comment (eliotis) [reply README.md]: Since schema is a Greek word, its plural would be schemata but for readability's sake we could use schema as plural just fine.

**Comment by eliotis** [comment]
> The changes look great to me! Incredible work @eliotis

Thanks @itstimetogoleft !

File: docs/interfaces.md [hunk 2]
```diff
@@ -4,15 +4,15 @@ title: Interfaces
...
-In object-oriented programming it is common to create interfaces which describes the contract that classes implementing them has to fulfill - hence TypeGraphQL supports defining GraphQL interfaces.
+In object-oriented programming it is common to create interfaces which describe the contract that classes implementing them must adhere to. Hence, TypeGraphQL supports defining GraphQL interfaces.
 
-## How to?
+## How to
 
 TypeScript has first class support for interfaces. Unfortunately, they only exist at compile-time, so we can't use them to build GraphQL schema at runtime by using decorators.
 
-Luckily, we can use an abstract class for this purpose. It behaves almost like an interface (can't be "newed", can be implemented by class), it just won't stop developers from implementing a method or initializing a field. But as long as we treat it like an interface, we can safely use it.
+Luckily, we can use an abstract class for this purpose. It behaves almost like an interface - it can't be "newed" but it can be implemented by the class - and won't prevent developers from implementing a method or initializing a field. So, as long as we treat it like an interface, we can safely use it.
```
Comment (19majkel94) [inline docs/interfaces.md]: > and won't prevent

It's not a feature but a drawback - we should distinct that, `it's just` was for that purpose.

File: docs/scalars.md [hunk 3]
```diff
@@ -88,14 +88,14 @@ class User {
   registrationDate: Date;
 }
 ```
-
-If you use `ts-node`, be aware you must execute it with the `--type-check` flag due to a [Date reflection bug](https://github.com/TypeStrong/ts-node/issues/511).
+Note:
```
Comment (19majkel94) [inline docs/scalars.md]: Basically it has been fixed, so I would ask you to delete this note by the way 😉

File: docs/faq.md [hunk 4]
```diff
@@ -4,50 +4,49 @@ title: Frequently Asked Questions
...
+This depends on various factors:
 
-- if your resolver need access only to the root/object value - use a getter
-- if your field has arguments
-  - and need to perform side effects (like db call) - use resolver class's method (leverage dependency injection mechanism)
-  - otherwise - use object type's methods (pure function, calculate based on object value and arguments)
-- if you want to separate business logic from type definition - use resolver class's method
+- if the resolver only needs access to the root/object value - use a getter
+- if the field has arguments
+  - and must perform side effects e.g. a database call - use a resolver class method and leverage the dependency injection      mechanism
```
[the remark this answers is not in the document]
Comment (eliotis) [reply docs/faq.md]: Must have been extra spaces I guess.

File: README.md [hunk 5]
```diff
@@ -118,27 +118,27 @@ type Mutation {
...
 
 The [Tests folder](https://github.com/19majkel94/type-graphql/tree/master/tests) might also give you some tips how to get various things done.
 
 ## Towards v1.0
 
-Currently released version is a MVP (Minimum Viable Product). It is well tested (96% coverage, 8000 lines of test code) and has 95% of the planned features already implemented.
+The currently released version is a MVP (Minimum Viable Product). It is well tested (96% coverage, 8000 lines of test code) and has 95% of the planned features already implemented.
 
-However there's some work to do before [1.0.0 release](https://github.com/19majkel94/type-graphql/milestone/3) and it's mostly about documentation (website, api reference and jsdoc) and compatibility with GraphQL spec or other tools.
+However there's still work to be before the [1.0.0 release](https://github.com/19majkel94/type-graphql/milestone/3) and it's mostly about documentation (website, api reference and jsdoc) and compatibility with the GraphQL spec and other tools.
```
[the remark this answers is not in the document]
Comment (eliotis) [reply README.md]: Correct, missed the word "done".

File: docs/faq.md [hunk 6]
```diff
@@ -59,16 +58,16 @@ updateItem(data: UpdateItemInput!): Item!
 updateItem(id: Int!, userId: Int!): Item!
 ```
 
-### When I have to use the `() => [ItemType]` syntax?
+### When should I use the `() => [ItemType]` syntax?
 
-You should use `[ItemType]` syntax every time when your field type is array or you return array from query/mutation.
+We should use the `[ItemType]` syntax any time the field type is an array or the return array from a query or mutation.
```
[the remark this answers is not in the document]
Comment (eliotis) [reply docs/faq.md]: Right, corrected.

File: docs/interfaces.md (hunk 2 above)
Comment (eliotis) [reply docs/interfaces.md]: OK added "it just" back in.

File: docs/scalars.md (hunk 3 above)
Comment (eliotis) [reply docs/scalars.md]: OK, deleted note.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (eliotis) [reply README.md]: Right, changed.

**Comment by eliotis** [comment]
Made requested changes/corrections, please let me know if there is anything else.

**Comment by 19majkel94** [comment]
@eliotis 
Thank you once again for the amazing work you've done ❤️

**Pull request merged**